### PR TITLE
add iputils

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

#### For security-related PRs

- [ ] The security fix is recorded in `advisories` and `secfixes`

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

**Review by dlorenc — APPROVED** [review]

File: iputils.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package:
+  name: iputils
+  version: '20221126'
```
Comment (dlorenc) [inline iputils.yaml:3]: Oh wait hmm... This might need to be 0_ prefixed.

**Review by dlorenc — DISMISSED** [review]

File: iputils.yaml (hunk 1 above)
Comment (dlorenc) [reply iputils.yaml:3]: Yeah I think we need to start this with 0.

https://github.com/wolfi-dev/os/blob/main/minio.yaml#L5

Comment (developer-guy) [reply iputils.yaml:3]: I just saw the argon.yaml which uses this syntax: _https://github.com/wolfi-dev/os/blob/main/argon2.yaml#L3_

So, should I fix also the argon package version?

Comment (developer-guy) [reply iputils.yaml:3]: I added '0.' prefix to the version.

Comment (developer-guy) [reply iputils.yaml]: this change caused an error like this: 
_https://github.com/wolfi-dev/os/actions/runs/5287579349/jobs/9568238586?pr=2853#step:5:10_

**Review by kaniini — APPROVED** [review]

**Comment by kaniini** [comment]
I think we should use the date as the version, it is consistent with the upstream version.  We only do the `0.` thing when there are no real "releases."

**Review by kaniini — CHANGES_REQUESTED** [review]
Please change the version to `20221126`.

**Review by kaniini — APPROVED** [review]

**Pull request merged**